src/js/components/Anchor/index.d.ts

```diff
@@ -15,6 +16,6 @@ export interface AnchorProps {
   as?: string;
 }
 
-declare const Anchor: React.ComponentClass<AnchorProps & JSX.IntrinsicElements['a']>;
+declare const Anchor: React.ComponentClass<AnchorProps & Omit<JSX.IntrinsicElements['a'], 'color'|'string'>>;
```

Comment:
I think these should just be `Omit<JSX.IntrinsicElements['a'], 'color'>`. There is no property called `string`.

Reply:
thanks for catching that.